crates/robonode-client/src/enroll.rs

```diff
@@ -280,7 +323,7 @@ mod tests {
             };
 
             let response =
-                ResponseTemplate::new(case.0).set_body_json(mkerr_containing_blob(case.1, "blob"));
+                ResponseTemplate::new(case.0).set_body_json(mkerr(case.1, "scan result blob"));
```

Comment:
It is possible that the current protocol has replies that don't have a blob, right?
In this regard, we should specify whether there is a blob or not in the response on a case-by-case basis.

To elaborate: *this* test fn should cover cases for both `FaceScanRejected` and `InvalidPublicKey`, but only `FaceScanRejected` has a blob in the response, while `InvalidPublicKey` has no blob.

Reply:
On the implementation note for this: we should add a new enum `ResponseIncludesBlob { Yes, No }` and add a corresponding value of it to each case's tuple. I.e. a `bool`.